packages/.gitignore

```
@@ -0,0 +1 @@
+**/*.patch.bz2
```

```suggestion
*.patch.bz2
```
I think this simpler version will work without having to include the wildcard for directories.